src/components/interaction-designer/block-types/ConsoleIO_PrintBlock.vue

```diff
@@ -5,9 +5,15 @@
     </h3>
 
     <fieldset :disabled="!isEditable">
-      <block-name-editor :block="block" />
-      <block-label-editor :block="block" />
-      <block-semantic-label-editor :block="block" />
+      <validation-message :message-key="`block/${block.uuid}/.name`" #input-control="{ isValid: isNameValid }">
+        <block-name-editor :block="block" :validationState="!isNameValid" />
```

Comment:
@poojahpatil90 , what if don't add the `<validation-message>` in this level, but under the `<block-....-editor>` component level ? We still have access to `:block` prop there, right ? This avoids us repetitive codes too.

I know you just picked the example I provided from `FlowEditor`, but now I'm wondering if there would be a best & cleaner approach.

Reply:
This would optimize our code and reduce redundant code. Made this change where ever it was possible